src/index.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import App from "./components/App.js";
+const rootEl = document.getElementById('app');
```

Comment:
Is this referring all the way back to public/index.html's div in the "real" dom?